test/app.calltracers.js

```diff
@@ -0,0 +1,36 @@
+
+var express = require('../');
+
+describe('app', function(){
+  describe('._callTracers(options)', function(){
```

Comment:
I wouldn't bother adding a test for an underscore-prefixed method, unless we're intending it to be a public interface.